analyzer/runtime/runtime.go

```diff
@@ -40,30 +40,21 @@ var _ analyzer.Analyzer = (*Main)(nil)
 // NewRuntimeAnalyzer returns a new main analyzer for a runtime.
 func NewRuntimeAnalyzer(
 	runtime common.Runtime,
-	sourceConfig *config.SourceConfig,
+	sdkPT *sdkConfig.ParaTime,
 	cfg *config.BlockBasedAnalyzerConfig,
+	sourceClient *source.RuntimeClient,
 	target storage.TargetStorage,
 	logger *log.Logger,
 ) (*Main, error) {
-	ctx := context.Background()
-
-	// Initialize source storage.
-	client, err := oasis.NewRuntimeClient(ctx, sourceConfig, runtime)
-	if err != nil {
-		logger.Error("error creating runtime client",
-			"err", err,
-		)
-		return nil, err
-	}
 	roundRange := analyzer.RoundRange{
 		From: uint64(cfg.From),
 		To:   uint64(cfg.To),
 	}
 	ac := analyzer.RuntimeConfig{
 		RuntimeName: runtime,
-		ParaTime:    sourceConfig.SDKParaTime(runtime),
+		ParaTime:    sdkPT,
```

Comment:
Not sure if I commented on this in a recent PR of Warren's ...
WDYT about naming both the struct attribute and the function param `RuntimeInfo` or `RuntimeMeta` or `RuntimeMetadata`? `sdkPT`:
  a) is a little cryptic,
  b) uses "paratime", which none of the indexer codebase does,
  c) uses "sdk" in the name, which IIRC made sense at some point to differentiate the sdk struct from our own internal metadata-like struct, but now just needlessly echoes the type name.

Reply:
:eyes:

Reply:
Good points; switched to `RuntimeMetadata`!